### PR TITLE
Add a generic, unknown thing type.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

File: virtual-things-adapter.js [hunk 1]
```diff
@@ -138,6 +138,52 @@ const smartPlug = {
...
+    },
+    {
+      name: 'stringProperty',
+      value: 'blah',
+      metadata: {
+        type: 'string'
+      }
+    },
+    {
+      name: 'bareNumberProperty',
```
Comment (benfrancis) [inline virtual-things-adapter.js]: numberProperty

File: virtual-things-adapter.js [hunk 2]
```diff
@@ -138,6 +138,52 @@ const smartPlug = {
...
+      value: 12,
+      metadata: {
+        type: 'number'
+      }
+    },
+    {
+      name: 'unitNumberProperty',
```
Comment (benfrancis) [inline virtual-things-adapter.js]: numberUnitProperty

File: virtual-things-adapter.js [hunk 3]
```diff
@@ -138,6 +138,52 @@ const smartPlug = {
...
+      value: 34,
+      metadata: {
+        type: 'number',
+        unit: 'metres'
+      }
+    },
+    {
+      name: 'fullNumberProperty',
```
Comment (benfrancis) [inline virtual-things-adapter.js]: numberUnitMinMaxProperty

File: virtual-things-adapter.js [hunk 4]
```diff
@@ -156,7 +202,7 @@ const dimmableLight = {
 };
 
 const VIRTUAL_THINGS = [
-  // thing,
+  thing,
```
Comment (hobinjk) [inline virtual-things-adapter.js]: Could you actually move it to the end of the list? The way I generated Thing ids is based on index so prepending will cause backwards incompatibility.

**Pull request merged**